notebooks/Notebook1.md

```diff
@@ -1,34 +1,61 @@
+
+Images of plan for data structure:
+
+![IMG_8536](https://user-images.githubusercontent.com/97985236/229011899-63f02c5f-6cd8-45f6-b733-30941eff8a9f.jpeg)
+
+![IMG_8537](https://user-images.githubusercontent.com/97985236/229011930-5868f4ad-4787-41de-8b9b-c768a75b8c80.jpeg)
+
 
 ## Questions
 
 **What is the most pressing issue for your project? What design decision do
 you need to make, what implementation issue are you trying to solve, or how
 are you evaluating your design and implementation?**
+My most pressing issue at this time is making sure that I have my repo set up correctly for java. As of right now I used a tutorial to set up a repo for all of my code and it runs. I am just not sure it is correct. The next problem will be working to build the bridge from my parser to the data structre aspect of my project. 
 
 **What questions do you have for your critique partners? How can they best help
 you?**
+I do not have any questions at this time.
 
 **How much time did you spend on the project this week? If you're working in a
 team, how did you share the work?**
+I spent about 9-12 hours this week on the project
 
 **Compared to what you wrote in your contract about what you want to get out of this
 project, how did this week go?**
+This week went really well I basically completed my goals for week one and week 2. 
```

Comment:
This all seems like you made good progress relative to your contract, I'm glad you were able to get running code!